pkg/controller/directvolumemigration/rsync.go

```diff
@@ -92,6 +98,15 @@ func (t *Task) ensureRsyncEndpoint() error {
 				getNodeHostnameAtRandom(hostnames),
 				t.getServiceType(),
 			)
+			blockEndpoint = svcendpoint.NewEndpoint(
```

Comment:
what if there are no block volumes in the plan? would we still create these endpoints / servers?

Reply:
Just the endpoints, not the servers. Not sure if I can determine here if block volumes exist or not.